tests/helm/test_git_sync_scheduler.py

```diff
@@ -0,0 +1,136 @@
+            {'name': 'GIT_SYNC_ROOT', 'value': '/git-root', 'value_from': None},
+            {'name': 'GIT_SYNC_DEST', 'value': 'test-dest', 'value_from': None},
+            {'name': 'GIT_SYNC_ADD_USER', 'value': 'true', 'value_from': None},
+            {'name': 'GIT_SYNC_WAIT', 'value': '66', 'value_from': None},
+            {'name': 'GIT_SYNC_MAX_SYNC_FAILURES', 'value': '70', 'value_from': None}])
+
+        self.assertEqual(git_sync_container.volume_mounts,
+                         [k8s.V1VolumeMount(name="dags", mount_path="/git-root")])
+
+    def test_ssh_params_added(self):
+        res = render_chart("GIT-SYNC", self.object_with_ssh_params,
+                           show_only=["templates/scheduler/scheduler-deployment.yaml"])
+        dep: k8s.V1Deployment = render_k8s_object(res[0], k8s.V1Deployment)
+        git_sync_container = dep.spec.template.spec.containers[1]
+        env_dict = [e.to_dict() for e in git_sync_container.env]
+        self.assertEqual(env_dict, [
+            {'name': 'GIT_SSH_KEY_FILE', 'value': '/etc/git-secret/ssh', 'value_from': None},
+            {'name': 'GIT_SYNC_SSH', 'value': 'true', 'value_from': None},
+            {'name': 'GIT_KNOWN_HOSTS', 'value': 'false', 'value_from': None},
+            {'name': 'GIT_SYNC_REV', 'value': 'HEAD', 'value_from': None},
+            {'name': 'GIT_SYNC_BRANCH', 'value': 'test-branch', 'value_from': None},
+            {'name': 'GIT_SYNC_REPO', 'value': 'https://github.com/apache/airflow.git',
+             'value_from': None},
+            {'name': 'GIT_SYNC_DEPTH', 'value': '1', 'value_from': None},
+            {'name': 'GIT_SYNC_ROOT', 'value': '/git', 'value_from': None},
+            {'name': 'GIT_SYNC_DEST', 'value': 'repo', 'value_from': None},
+            {'name': 'GIT_SYNC_ADD_USER', 'value': 'true', 'value_from': None},
+            {'name': 'GIT_SYNC_WAIT', 'value': '60', 'value_from': None},
+            {'name': 'GIT_SYNC_MAX_SYNC_FAILURES', 'value': '0', 'value_from': None}])
+
+    def test_adds_git_username(self):
+        res = render_chart("GIT-SYNC", self.object_with_username,
+                           show_only=["templates/scheduler/scheduler-deployment.yaml"])
+        dep: k8s.V1Deployment = render_k8s_object(res[0], k8s.V1Deployment)
+        git_sync_container = dep.spec.template.spec.containers[1]
+        env_dict = [e.to_dict() for e in git_sync_container.env]
+        self.assertEqual(env_dict, [
+            {'name': 'GIT_SYNC_USERNAME', 'value': None, 'value_from':
+                {'config_map_key_ref': None,
+                 'field_ref': None,
+                 'resource_field_ref': None,
+                 'secret_key_ref': {'key': 'GIT_SYNC_USERNAME',
+                                    'name': 'user-pass-secret', 'optional': None}}},
+            {'name': 'GIT_SYNC_PASSWORD', 'value': None,
+             'value_from':
+                 {'config_map_key_ref': None,
```

Comment:
Can you enable black for this directory? I think this could significantly improve the readability of the tests. We try to enable black for the new code, eg airflow.providers, airflow.api_connexion.